sdk/python/feast/on_demand_feature_view.py

```diff
@@ -67,13 +69,13 @@ class OnDemandFeatureView(BaseFeatureView):
     owner: str
 
     @log_exceptions
-    def __init__(
+    def __init__(  # noqa: C901
```

Comment:
because of the branching logic since we're trying to maintain backwards compatibility flake8 is complaining that the init function for odfv is too complicated, henc the lint failure in github actions.